assembl/static2/js/app/components/debate/common/post/postBody.jsx

```diff
@@ -77,22 +77,29 @@ const Html = (props) => {
       html = html.body;
     }
     extracts.forEach((extract) => {
-      const wrapper = jQuery(`<annotation id="${extract.id}"></annotation>`);
-      extract.textFragmentIdentifiers.forEach((tfi) => {
-        const range = new ARange.SerializedRange({
-          start: tfi.xpathStart,
-          startOffset: tfi.offsetStart,
-          end: tfi.xpathEnd,
-          endOffset: tfi.offsetEnd
-        });
-        try {
-          const normedRange = range.normalize(html);
-          const nodes = jQuery(normedRange.textNodes()).filter((idx, node) => !white.test(node));
-          nodes.wrap(wrapper);
-        } catch (error) {
-          console.error(error); // eslint-disable-line no-console
+      if (extract) {
+        const tfis = extract.textFragmentIdentifiers;
+        const wrapper = jQuery(`<annotation id="${extract.id}"></annotation>`);
+        if (tfis) {
+          tfis.forEach((tfi) => {
+            if (tfi && tfi.xpathStart && tfi.offsetStart && tfi.xpathEnd && tfi.offsetEnd) {
+              const range = new ARange.SerializedRange({
+                start: tfi.xpathStart,
+                startOffset: tfi.offsetStart,
+                end: tfi.xpathEnd,
+                endOffset: tfi.offsetEnd
+              });
+              try {
+                const normedRange = range.normalize(html);
+                const nodes = jQuery(normedRange.textNodes()).filter((idx, node) => !white.test(node));
+                nodes.wrap(wrapper);
+              } catch (error) {
+                console.error(error); // eslint-disable-line no-console
```

Comment:
Do you only want the error to show in the console ? no displayAlert ?

Reply:
TODO: Adapt Raven to V2 so that instead of console errors, we can start sending front-end errors to Sentry...